### PR TITLE
Frsca Code of Conduct

### PR DESCRIPTION
**Pull request by anniepamela2020**
issue #312

### CODE REVIEW

File: .github/Code_of_conduct [hunk 1]
```diff
@@ -0,0 +1,128 @@
+# Contributor Covenant Code of Conduct
+
```
Comment (sudo-bmitch) [inline .github/Code_of_conduct:2]:
```suggestion

FRSCA is an OpenSSF project and follows their code of conduct:
https://openssf.org/community/code-of-conduct/

```

File: .github/Code_of_conduct [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
Comment (sudo-bmitch) [inline .github/Code_of_conduct:63]: Note to update this.

**Review by sudo-bmitch — COMMENTED** [review]
(Just a comment to avoid blocking the merge while I'm out.)
This is looking great. I would rename from `./.github/Code_of_conduct` to `./CODE_OF_CONDUCT.md` (placing it in the root of the repo, and formating as markdown). We should also include a link in the README.md under the Community section.

With DCO, we also want to add a contributor policy. Something like https://github.com/stolostron/policy-collection/blob/main/CONTRIBUTING.md (found on the first page of google results, so not a personal recommendation, but doesn't look bad to me).

**Comment by anniepamela2020** [comment]
will open a new PR

**Pull request closed without merging**